### ISSUE TITLE
Authentication flow to get a token no longer works

### ISSUE THREAD

**Issue by catichenor** [O365/python-o365#462]
A recent server-side authentication error appears to have invalidated all of the API access tokens that we have at my company, so I need to generate a new one as noted in the documentation at https://o365.github.io/python-o365/latest/html/usage/account.html#authenticating-your-account . 

Following the instructions in the docs, the page to authenticate no longer has the URL in the correct format. There is a note on that page that "If the url is simply https://outlook.office.com/owa/?realm=blahblah, and nothing else after that.. then you are currently on new Outlook look, revert back to old look and try the authentication flow again". This appears to be the case. But, the old Outlook look is [no longer available as of September 2019](https://answers.microsoft.com/en-us/msoffice/forum/all/revert-to-previous-version-of-officeoutlook-365/c0662da4-3d7f-4343-b148-833fccba8dbc). 

Is there any way to work around this?

**Comment by janscas** [comment]
Follow the instructions on the readme. Those are far more updated than the docs.

Docs need a serious update.

**Comment by catichenor** [comment]
Thanks janscas, I updated my version of the python-o365 library and now I'm receiving a different error than before when attempting to visit the consent URL. That error is "AADSTS50011: The reply URL specified in the request does not match the reply URLs configured for the application:". I tried generating a new client secret, as well as registering a new application/client ID, but neither of those worked. 

I suspect that this is a problem on the server side, but I'll keep this open until I can get this resolved, either through escalating an IT ticket or by trying a different authentication method.
